docs/command-replacements.md

```diff
@@ -21,11 +21,18 @@ The following values are replaced with the specific values at runtime.
 
 Assume the following:
 
-* `supervisor_options` is: "a", "{target_options}", "b"
-* `target_options` is: "c", "{target_exe}"
-* `target_exe` is: "d"
+* `supervisor_options` is: ``"a", "{target_options}", "b"`
+* `target_options` is: `"c", "{target_exe}"`
+* `target_exe` is: `"d"`
+
+The resulting `supervisor_options` is: `"a", "c d", "b"`.
+
+If you need `supervisor_options` to expand to: `"a", "c", "b", "d"`, you should use the following values:
+
+* `supervisor_options`: `"a", "c", "{target_exe}"`
```

Comment:
```suggestion
* `supervisor_options`: `"a", "c", "b", "{target_exe}"`
```

Reply:
You're right in that the logic in the original PR is wrong.  I've updated the content to use A, B, C, D in alphabetic ordering to reduce confusion.  Can you re-review?

Reply:
I thought you had a non alphabetic order as a trick to force the reader to pay attention.
But i am ok with the alphabetical order

Reply:
Nope.  I didn't think about readability of the example when I wrote it.  Your feedback made me recognize that it was hard to grok.  The alphabetic example is less difficult to trace.